flink-connectors/flink-connector-kinesis/pom.xml

```diff
@@ -33,7 +33,7 @@ under the License.
 	<artifactId>flink-connector-kinesis_${scala.binary.version}</artifactId>
 	<name>flink-connector-kinesis</name>
 	<properties>
-		<aws.sdk.version>1.11.171</aws.sdk.version>
+		<aws.sdk.version>1.11.250</aws.sdk.version>
```

Comment:
AFAIK, the exact `aws.sdk.version` was picked according to match the KCL / KPL dependency versions.
i.e., both the AWS Java SDK dependency version used in the KCL / KPL matches the `aws.sdk.version`, otherwise there may be conflicts.

Was `1.11.250` confirmed to also be compatible with the current KCL / KPL versions?